h2d/RenderContext.hx

```diff
@@ -249,14 +276,16 @@ class RenderContext extends h3d.impl.RenderContext {
 			absY = obj.absY;
 		}
 
+		if( !beginDraw(obj, tile.getTexture(), true, true) ) return false;

```

Comment:
We do not want to move this before checking out of screen, because this might cause shader switching for out of screen elements (move it back after the check, does it change something?)